gpytorch/beta_features.py

```diff
@@ -26,6 +26,25 @@ def __getattr__(self, name):
 fast_pred_samples = _moved_beta_feature(_fast_pred_samples)
 
 
+class checkpoint_kernel(_value_context):
```

Comment:
I know we used this for the existing code, but could we change to e.g. `checkpoint_kernel_size` or `kernel_partition_size` or anything that has the word "size" in it since that's what the value represents?